client.go

```diff
@@ -79,11 +79,8 @@ func (e *APIError) Unwrap() error {
 	return e.e
 }
 
-type InvalidCredentialsError struct{}
-
-func (e InvalidCredentialsError) Error() string {
-	return "invalid credentials"
-}
+var InvalidCredentialsError = fmt.Errorf("invalid credentials")
+var InvalidCodeError = fmt.Errorf("invalid enrollment code")
```

Comment:
i know you were just following the existing code, but these should be in the format `ErrInvalidCredentials` and `ErrInvalidCode`.

Since we'll have to update callers anyway (errors.As -> errors.Is for InvalidCredentialsError), I think we should fix the names now.

Reply:
done